next-14/next.config.mjs

```diff
@@ -1,4 +1,9 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
-
-export default nextConfig;
+const nextConfig = {
+    images: {
+        domains: ['res.cloudinary.com'],
+      },   
+ };
```

Comment:
Julee comments. 

They wanted to add 
async rewrites() {
    return [
      {
        source: '/api/protected-route',
        destination: '/api/protected-route',
      },
    ];
  },
  
  but since that doesn't exist in our template I'm not sure we should add it.

Reply:
was there a reason to add this ? That rewrite looks like a NOOP

Reply:
I can remove it if needed, it didn't really make a difference and Julee put it in their mini tutorial: https://dev.to/userfront/security-charging-your-nextjs-14-app-with-userfront-authentication-2knc

So I kept it